src/app/lib/config/services/amharic.js

```diff
@@ -45,29 +45,25 @@ const service = {
     error: {
       404: {
         statusCode: '404',
-        title: '404 - ገጹ አልተገኘም',
-        message:
-          'ይህ የድረገጽ አድራሻውን በትክክል ስላላስገቡ ሊሆን ይችላል፡፡ እባክዎትን አድራሻውን እና ፊደላቱን ያረጋግጡ፡፡',
+        title: '404 - ገፁን አሁን ማግኘት አይችሉም',
+        message: 'የሚፈልጉትን ገፅ ማምጣት ባለመቻላችን እናዝናለን። እባክዎ እንደገና ይሞክሩ',
         solutions: [
-          'Double checking the url',
-          'Hitting the refresh button in your browser',
-          'Searching for this page using the BBC search bar',
+          'ድረገፁን በድጋሚ በመሞከር ላይ',
+          'ገፅዎ ላይ ያለውን ሪፍሬሽ ቁልፍ  እንደገና ተጭነው ይሞክሩት',
+          'የቢቢሲን መፈለጊያ ቁልፍ በመጠቀም ይፈልጉ',
         ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'የቢቢሲ ዜና መነሻ ገጽ',
+        callToActionFirst: 'እንደ አማራጭም እባክዎ ገፃችን ላይ የሚገኘውን',
+        callToActionLinkText: 'BBC ድረገፅ ይጎብኙ',
         callToActionLast: '',
         callToActionLinkUrl: 'https://www.bbc.com/amharic',
       },
       500: {
         statusCode: '500',
-        title: '500 - ስህተት',
-        message: 'ስህተት ተፈጥሯል ፡ እባክዎን ገጽዎን ያድሱ',
-        solutions: [
-          'Hitting the refresh button in your browser',
-          'Coming back again later',
-        ],
-        callToActionFirst: 'Alternatively, please visit the ',
-        callToActionLinkText: 'የቢቢሲ ዜና መነሻ ገጽ',
+        title: '500 - የአገልግሎት ችግር አጋጥሟል',
```

Comment:
Same as the comment above - to remove the `500 - ` part of this string.

Reply:
Thanks Jonathan